src/client/render.tsx

```diff
@@ -64,7 +66,7 @@ export class CellOutput extends React.Component<ICellOutputProps> {
         const imgSrc =
             mimeType.toLowerCase().includes('svg') && typeof data === 'string' ? undefined : URL.createObjectURL(data);
         const customMetadata = metadata.metadata as JSONObject | undefined;
-
+        const showPlotViewer = metadata.__displayOpenPlotIcon === true;
```

Comment:
Seems weird that the renderer knows about metadata from the extension. Feels like the code adding this metadata should be in the same extension. I wonder if we can handle output in this extension and add the metadata tag?

Reply:
Maybe using this?
https://github.com/microsoft/vscode/blob/c1ae18fad7924af3f4bf4171b8c269b52ce59e21/src/vs/vscode.proposed.d.ts#L1171

Reply:
> I wonder if we can handle output in this extension and add the metadata tag?

Then the renderer needs to be in the same extension. I think we need commands for outputs (that would solve this)

Reply:
I think this will happen https://github.com/microsoft/vscode/issues/124917

Reply:
> Then the renderer needs to be in the same extension

Yeah that's kinda my point. The ```__displayOpenPlotIcon``` is an implicit dependency right now. Seems better if the renderer extension had this in in its extension.js file (although that's a moot point at the moment, but might not be soon).

Commands for outputs would eliminate the need for this code altogether then. That would be cool.